---
title: "The sweet cherry genome: A new sequencing project"
collection: talks
type: "Poster"
permalink: /talks/2017-The-sweet-cherry-genome
date: 2017-01-14 - 2017-01-18
venue: 'Plant and Animal Genome XXV'
paperurl: 'https://pag.confex.com/pag/xxv/meetingapp.cgi/Paper/25535'
location: "San Diego, United States"
---
Authors: Mathieu Fouche, Loick Le Dantec, Bénédicte Wenden, José Quero-Garcia, Teresa Barreneche, Ksenija Gasic, Goran Barac, Christopher A. Saski, Amy Lezzoni, Felicidad Fernandez, Herman Silva, Elisabeth Dirlewanger

<i class="ai ai-open-access"></i> [Download poster here](https://pag.confex.com/pag/xxv/mediafile/Handout/Paper25535/poster-genome%20pag.pdf) (PDF, 482 KB)

**Abstract**: The sweet cherry (<i>Prunus avium</i>) is a diploid species with an estimated genome size of 338 Mb Arumuganathan et al (1991). It is mostly self-incompatible, and therefore has a heterozygous genetic background. Through a sweet cherry international consortium recently created, the first sweet cherry reference genome is in progress. To do this, we chose the“Regina” variety: it is a late blooming cultivar from which a unigene and transcriptomic data are available. This data will be useful for the future annotation. Moreover, it is the parent of two segregating populations used to build saturated genetic maps, which will help the scaffolding process. The long reads sequencing technology (PacBIO RSII) was used in order to produce a robust and high quality <i>de novo</i> genome. Indeed, the long reads sequencing allows a significant smaller number of contigs and goes through repeat or low-complexity regions. We performed a DNA extraction following the protocol published by Mayjonade et al (2016) that we customized for sweet cherry to produce a high purity and high molecular weight DNA needed for the PacBIO sequencing. A depth of 82X was obtained with a N50 of 17.5Kb. The de novo assembly of “Regina” will be done using a combination of long reads assembly tools (Falcon, CANU,…) . Moreover, an optical mapping is in progress to obtain a more accurate and contiguous genome.
